### PR TITLE
Beautify the YOLOv5 configuration

### PR DESCRIPTION
**Pull request by hhaAndroid**
Thanks for your contribution and we appreciate it a lot. The following instructions would make your pull request more healthy and more easily get feedback. If you do not understand some items, don't worry, just make the pull request and seek help from maintainers.

## Motivation

Please describe the motivation for this PR and the goal you want to achieve through this PR.

## Modification

Please briefly describe what modification is made in this PR.

## BC-breaking (Optional)

Does the modification introduce changes that break the backward compatibility of the downstream repos?
If so, please describe how it breaks the compatibility and how the downstream projects should modify their code to keep compatibility with this PR.

## Use cases (Optional)

If this PR introduces a new feature, it is better to list some use cases here and update the documentation.

## Checklist

1. Pre-commit or other linting tools are used to fix potential lint issues.
2. The modification is covered by complete unit tests. If not, please add more unit tests to ensure the correctness.
3. If the modification has a potential influence on downstream projects, this PR should be tested with downstream projects, like MMDetection or MMClassification.
4. The documentation has been modified accordingly, like docstring or example tutorials.

### CODE REVIEW

File: configs/yolov5/yolov5_s-v61_syncbn_8xb16-300e_coco.py [hunk 1]
```diff
@@ -1,47 +1,95 @@
...
+weight_decay = 0.0005
+# Save model checkpoint and validation intervals
+save_epoch_intervals = 10
+# The maximum checkpoints to keep.
+max_keep_ckpts = 3
+# Single-scale training is recommended to
 # be turned on, which can speed up training.
 env_cfg = dict(cudnn_benchmark=True)
 
+# ==========================================================================================
```
Comment (RangeKing) [inline configs/yolov5/yolov5_s-v61_syncbn_8xb16-300e_coco.py:92]:
```suggestion
# ========================Infrequently modified parameters=============================
```

Comment (hhaAndroid) [reply configs/yolov5/yolov5_s-v61_syncbn_8xb16-300e_coco.py:92]: 需要改成这样吗，我那个意思是这里就是结尾

Comment (RangeKing) [reply configs/yolov5/yolov5_s-v61_syncbn_8xb16-300e_coco.py:92]: 万一用户想改下面的内容的话，感觉这个分割线以下的参数最好也有个标题说明会好一点（不一定要写成这个标题

Comment (hhaAndroid) [reply configs/yolov5/yolov5_s-v61_syncbn_8xb16-300e_coco.py]: OK

File: configs/yolov5/yolov5_s-v61_syncbn_8xb16-300e_coco.py [hunk 2]
```diff
@@ -1,47 +1,95 @@
...
-img_scale = (640, 640)  # width, height
-deepen_factor = 0.33
-widen_factor = 0.5
-max_epochs = 300
-save_epoch_intervals = 10
+num_classes = 80  # Number of classes for classification
+# Batch size of a single GPU during training
 train_batch_size_per_gpu = 16
+# Worker to pre-fetch data for each single GPU during training
 train_num_workers = 8
```
Comment (RangeKing) [inline configs/yolov5/yolov5_s-v61_syncbn_8xb16-300e_coco.py:17]: `train_batch_size_per_gpu` and `train_num_workers` should be moved into `# -----train val related-----`?

Comment (hhaAndroid) [reply configs/yolov5/yolov5_s-v61_syncbn_8xb16-300e_coco.py:17]: 这个是data相关参数，所以我就放到data 里面去了

Comment (RangeKing) [reply configs/yolov5/yolov5_s-v61_syncbn_8xb16-300e_coco.py:17]: ok

**Review by RangeKing — APPROVED** [review]
LGTM

**Pull request merged**